serenity-junit5/pom.xml

```diff
@@ -39,8 +39,25 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.junit</groupId>
+            <artifactId>junit-bom</artifactId>
+            <version>${junit5.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
```

Comment:
Are you sure this is needed since it is available in the ParentPom to avoid to define junit5 artifacts version?

Reply:
Yes, you are right @DanielFran - I removed that and all maven targets still ran okay 👍 - repushed and also added in some missing files that mysteriously were missing on last push 🤦